elogia_hr/models/hr.py

```diff
@@ -103,10 +97,10 @@ def write(self, vals):
                 if value_parent != record.parent_id:
                     list_field.append({'value_before': record.parent_id.name, 'value_actual':
                         value_parent.name, 'type_action': 'responsible', 'type_model': type_model, 'employee_id': record.id})
-            if vals and 'hub_id' in vals:
-                value_hub = hub_env.search([('id', '=', vals['hub_id'])], limit=1)
-                if value_hub != record.hub_id:
-                    list_field.append({'value_before': record.hub_id.name, 'value_actual':
+            if vals and 'default_planning_role_id' in vals:
```

Comment:
Solo como observación : por que preguntar por if vals en "if vals and 'default_planning_role_id' in vals:" Si vals estuviera vacío simplemente te devolvería falso si no contiene la key, la condición podría ser solamente "if 'default_planning_role_id' in vals:"